CRP.Mvc/App_Start/BundleConfig.cs

```diff
@@ -14,8 +14,12 @@ public static void RegisterBundles(BundleCollection bundles)
             bundles.Add(new ScriptBundle("~/bundles/jqueryCurrent").Include(
                         "~/Scripts/jquery-1.10.2.js"));
 
-            bundles.Add(new ScriptBundle("~/bundles/jqueryval").Include(
-                        "~/Scripts/jquery.validate*"));
+            //Needs <script src="https://ajax.microsoft.com/ajax/jquery.validate/1.8.1/jquery.validate.min.js"></script>
+            bundles.Add(new ScriptBundle("~/bundles/checkout").Include(
+                        "~/Scripts/xVal.jquery.validate.js",
```

Comment:
@jSylvestre we're not really using xVal are we?  That has been deprecated for half a decade and their website says don't use it with anything after MVC 1.